config/crds/topology.kcp.io_partitionsets.yaml

```diff
@@ -66,11 +65,26 @@ spec:
                         key:
                           description: key is the label key that the selector applies
                             to.
+                          maxLength: 317
                           type: string
+                          x-kubernetes-validations:
+                          - message: Label key prefixes are optional and limited to
```

Comment:
I have no way of verifying this validation, I think we will validate it in an e2e test.

Reply:
yes I will add checks for the admission to the e2e PR